src/gsy_e/models/market/grid_fees/base_model.py

```diff
@@ -66,7 +66,7 @@ def update_forwarded_bid_trade_original_info(self, trade_original_info, market_b
         if not trade_original_info:
             return None
         trade_offer_info = TradeBidOfferInfo(
-            original_bid_rate=market_bid.original_price / market_bid.energy,
+            original_bid_rate=market_bid.original_energy_rate,
```

Comment:
nice! this will save also some calculation effort, right?

Reply:
Unfortunately not, because this is a property. We will not have to remind ourselves to do the rounding though, because it is handled by the property.

Reply:
I saw this now in the framework PR. Thanks